mmv1/templates/terraform/examples/sql_postgres_instance_backup.tf.erb

```diff
@@ -0,0 +1,19 @@
+provider "google" {
+ credentials = file("CREDENTIALS_FILE.json")
```

Comment:
Is this meant to be in this example only? The test is failing because there is no `CREDENTIALS_FILE.json` in our test directory. also we wouldn't want to override our test credentials anyway.

If we want a "documentation-only" example, we can add `skip_test: true` to the example block in the `terraform.yaml`

Reply:
Removed